client/src/game/ui/reactive/view/cardsView.ts

```diff
@@ -110,6 +110,19 @@ const subscribeToCardChanges = (order: number) => {
     location: c.location,
   }), () => updateBorder(order));
 
+  // Card fade and critical indicator
+  subFullState((s) => {
```

Comment:
if it only uses visible state, then shouldnt the sub be to the visible state?

Reply:
It's literally the same

Reply:
the "sub visible" is a shortcut for what he did

Reply:
if its the same, then use the provided helper and put the code in the correct section

Reply:
The helper is not in this file.

Reply:
ok never mind then